vars/liskBuild.groovy

```diff
@@ -0,0 +1,47 @@
+
+
+def call(Map params) {
+	def liskBranch = params.branch
+	def buildNetwork = params.network
+
+	def liskDir = "lisk-${liskBranch}"
+	def liskVersion = ""
+	def liskReleaseName = ""
+
+	def liskBuildDir = ""
+	def liskBuildBranch = ""
+	def	latestVersion = false 
+
+	dir(liskDir) {
+		git url: "https://github.com/LiskHQ/lisk.git", branch: liskBranch
+
+		sh """
+		npm install
+		node ./node_modules/.bin/grunt release
+		jq -r '.version' config.json > ".lisk-version"
+		"""
+	}
+	script {
+		liskVersion = readFile("${liskDir}/.lisk-version").trim()
+
+		if(fileExists("${liskDir}/genesisBlock.json")) {
+			liskBuildBranch = "0.9.15"
```

Comment:
I am a little worried about this: how will we make sure this does not get forgotten / gets updated in the future?
Can we not use `master`?

Reply:
You should ask @Nazgolze for it, as he is the source of truth for this stuff. Once he merged the branches accordingly we will update our code as well. BTW after 1.0.0 release every legacy branch can be archived.

Reply:
Agreed, we should keep it as less hardcoded as we can. I think we can just assume the new versioning / naming.